webui/src/main/java/org/polypheny/db/webui/models/requests/EditCollectionRequest.java

```diff
@@ -34,7 +34,7 @@ public class EditCollectionRequest {
     public String action; // truncate / drop
 
     /**
-     * Identifier of the store on which the collection is edited
+     * Identifier of the storeId on which the collection is edited
```

Comment:
id is the identifier. So the comment was correct, wasn't it?

Reply:
Yes, adjusted.